### PR TITLE
Release: merge development into main

### PR DESCRIPTION
**Pull request by dimileeh**
Automated AWF release PR syncing `development` into `main`.

Opened by the `sync_release_pr` task kind and monitored with release/manual behavior (auto-merge disabled). Merging into the release target stays human-gated.

---

> [!NOTE]
> **Medium Risk**
> Changes PR monitor merge-gating and persisted verdict state when operator hints finalize; incorrect id matching could clear or retain the wrong human wait, though guards (body hash, contextual bare ids) limit blast radius.
> 
> **Overview**
> **Operator hint finalization** now clears stale review-level `needs_human` waits when a consumed guide’s **acted-on** text (directive, or remonitor `reason` only when the CLI actually ran) names matching feedback keys (`issue:…`, `bbcomment:…`, or contextual bare ids). Parsing avoids accidental clears from unrelated long numbers; retirement requires an existing body-hash sidecar and leaves unrelated comments blocking.
> 
> **Resume paths** thread `hint` and `acted_feedback_text` into `_finalize_operator_hint_resume` so preserved-head **restart shortcuts** that skip the CLI do not treat audit/remonitor reason text as answered feedback.
> 
> **CI** replaces `uv venv` + `uv pip install -e ".[dev]"` with **`uv sync --extra dev`** and a **3-attempt retry** loop on transient install failures.
> 
> Tests cover id matching edge cases, `decide()` merge vs `NotifyHuman`, and remonitor shortcut behavior; some preserved-head tests move to a new part file.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 63914640057812bfe38f41a8b509f10aa173c580. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

**Comment by dimileeh** [comment]
⚠️ PR #686 needs human attention at commit `d04deadcdc`.

AWF did not auto-merge because a protected quality-gate file was changed and AWF paused the workspace for an operator decision: agent changed protected quality-gate file(s) outside declared owned_paths:
- .github/workflows/ci.yml :: jobs.lint-and-type.steps.Install dependencies.run :: line 42 :: workflow run command changed outside informational step: jobs.lint-and-type.steps.Install dependencies.run
AWF only allows narrowly classified safe protected-file edits without explicit ownership. Add meaningful tests or declare explicit ownership of the policy file when an operator-approved policy change is required..

After the blocker is cleared or a new commit lands, AWF will re-verify the PR before taking any merge action.

File: src/awf/runtime/pr_monitor_runner/operator_hints.py [hunk 1]
```diff
@@ -41,6 +42,28 @@
...
+_OPERATOR_HINT_BARE_FEEDBACK_ID_RE = re.compile(
+    r"""
+    \b
+    (?:
+        feedback
+        | review(?:[\s_-]+comment)?
+        | comment
+    )
+    [\s_-]*id[\s:#-]*
+    (?P<id>\d{6,})
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/runtime/pr_monitor_runner/operator_hints.py]: Fixed in local commit `639146400`.

What changed:
- Contextual bare feedback IDs now accept short digit strings like `comment id 55`.
- Short bare IDs retire only the exact bare key, so an unrelated `issue:55` wait is not cross-mapped.
- Added regression coverage for `comment id 55` preserving that exact-key behavior.

Targeted checks:
- `uv run --python 3.12 --extra dev pytest tests/unit/runtime/test_pr_monitor_operator_hint_coverage_edges.py::test_operator_hint_retire_referenced_needs_human_feedback_accepts_short_bare_id -q` failed before the implementation, as expected.
- `uv run --python 3.12 --extra dev pytest tests/unit/runtime/test_pr_monitor_operator_hint_coverage_edges.py -q` passed: 43 passed.

I did not modify `.github/workflows/ci.yml`.

**Comment by dimileeh** [comment]
✅ PR #686 is ready to merge at commit `6391464005`.

All 5 AWF gates are green:
1. Inline comments resolved.
2. Outside-diff comments addressed.
3. CI checks all SUCCESS or SKIPPED.
4. Mergeable.
5. Base merged into head.

AWF will not merge this PR automatically — human action required. If new commits land here, AWF will re-verify all 5 gates and re-post this message on the new head SHA.
